administrator/language/de-DE/com_cpanel.ini

```
@@ -24,13 +24,13 @@ COM_CPANEL_MSG_LANGUAGEACCESS340_BODY="Seit Joomla! 3.4.0 gibt es möglicherweis
 COM_CPANEL_MSG_ROBOTS_TITLE="Änderungen an der Standard robots.txt-Datei"
-COM_CPANEL_MSG_STATS_COLLECTION_BODY="<p>Seit Joomla! 3.5 sendet ein Statistik-Plugin anonyme Daten zum Joomla!-Projekt. Dabei werden nur die Joomla!- und PHP-Version, der Datenbanktyp und -version und das Serverbetriebssystem gesendet.</p><p>Diese Daten werden für zukünftige Joomla!-Versionen erhoben, um sicherzustellen, dass diese von den neusten Datenbank- und PHP-Funktionen profitieren können, ohne das vorhandene Joomla!-Benutzer Probleme bekommen. Die Notwendigkeit wurde klar, als als Minimum PHP 5.3.10 für Joomla! 3.3 notwendig wurde, da hier die sichere Passwortspeicherung mit Bcrypt eingeführt wurde.</p><p>Im Interesse voller Transparenz und um Erweiterungsentwicklern zu helfen, sind die <a href='https://developer.joomla.org/about/stats.html' target='_blank' rel='noopener noreferrer'>Daten öffentlich einsehbar.</a> Eine API und Diagramme zeigen die verwendeten Joomla!-Versionen, PHP-Versionen und Datenbanktypen.</p><p>Wenn das Senden dieser Daten an das Joomla!-Projekt unerwünscht ist, kann das Plugin „System – Joomla!-Statistikerhebung“ deaktiviert werden.</p>"
+COM_CPANEL_MSG_STATS_COLLECTION_BODY="<p>Seit Joomla! 3.5 sendet ein Statistik-Plugin anonyme Daten zum Joomla! Projekt. Dabei werden nur die Joomla- und PHP-Version, der Datenbanktyp und -version und das Serverbetriebssystem gesendet.</p><p>Diese Daten werden für zukünftige Joomla-Versionen erhoben, um sicherzustellen, dass diese von den neusten Datenbank- und PHP-Funktionen profitieren können, ohne das vorhandene Joomla-Benutzer Probleme bekommen. Die Notwendigkeit wurde klar, als als Minimum PHP 5.3.10 für Joomla! 3.3 notwendig wurde, da hier die sichere Passwortspeicherung mit Bcrypt eingeführt wurde.</p><p>Im Interesse voller Transparenz und um Erweiterungsentwicklern zu helfen, sind die <a href='https://developer.joomla.org/about/stats.html' target='_blank' rel='noopener noreferrer'>Daten öffentlich einsehbar.</a> Eine API und Diagramme zeigen die verwendeten Joomla-Versionen, PHP-Versionen und Datenbanktypen.</p><p>Wenn das Senden dieser Daten an das Joomla! Projekt unerwünscht ist, kann das Plugin „System – Joomla-Statistikerhebung“ deaktiviert werden.</p>"
 COM_CPANEL_MSG_STATS_COLLECTION_TITLE="Statistikerhebung in Joomla"
```

```suggestion
COM_CPANEL_MSG_STATS_COLLECTION_TITLE="Statistikerhebung in Joomla!"
```
see Comment on line R97 file administrator/language/de-DE/com_config.ini